### PR TITLE
Remove unecessary dependency "shortid"

### PR DESCRIPTION
**Pull request by GeorgeTaveras1231**
I want to show a path for generating unique IDs without requiring a dependency.

Additionally, I updated the `Field` `Dropdown` and `Textarea` component to use the same method for a unique id.

### CODE REVIEW

File: packages/@xo-union/tk-component-fields/src/utilities/index.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
   return string.split(/[\W\s]/g).map(word =>
     word.replace(/^(\w)/, w => w.toUpperCase())
   ).join(' ');
 }
 
-export function fieldId(name) {
-  return `__ff-${name}__`;
+let idCounter = 0;
+export function fieldId() {
+  return `__UNION_FIELD__${__webpack_hash__}_${idCounter++}__`;
```
Comment (GeorgeTaveras1231) [inline packages/@xo-union/tk-component-fields/src/utilities/index.js]: @marioharper This being the magic sauce that generates unique IDs. It uses the webpack hash which will be unique per package version. And using an incrementing counter.

**Comment by GeorgeTaveras1231** [comment]
History needs to be cleaned since #276 was merged, which was the original base for the PR

**Comment by marioharper** [comment]
@GeorgeTaveras1231 is the webpack hash necessary?

**Comment by GeorgeTaveras1231** [comment]
@marioharper Yes to account for the edge case where two versions of this same package make it into the same bundle. Something like

```
my-app
|___ @xo-union/tk-component-membership-modal@1.0.0
|     |___@xo-union/tk-component-fields@2.0.0
|___@xo-union/tk-component-fields@1.0.0
```

**Comment by GeorgeTaveras1231** [comment]
@marioharper otherwise, a counter would do. But because the counter is instantiated per instance of the module, it would be problematic with multiple instances

**Pull request merged**